docs/reference/target-frameworks.md

```diff
@@ -85,6 +85,7 @@ Universal Windows Platform | uap | uap [uap10.0] |
 | | | netcoreapp2.2 |
 | | | netcoreapp3.0 |
 | | | netcoreapp3.1 |
+.NET | net | net5.0 |
```

Comment:
This is not quite right. .NET 5 is the successor to netcoreapp3.1(i.e. net5.0 for short). Thus we should mimic the table found here: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-frameworks

Here's a tool to show the precedence for reference!

https://nugettoolsdev.azurewebsites.net/5.8.0/framework-precedence?framework=net5.0&exludedIdentifiers=

If you'd like to update your PR to reflect that, we can merge! Thanks in advance!

Reply:
Thank you, Done.
https://github.com/NuGet/docs.microsoft.com-nuget/pull/2398/commits/ff1ef787a30a368bebb3407ac7f0711340cd2094